movieapp/src/services/utils.ts

```diff
@@ -5,5 +5,5 @@ type SearchParams = {
 };
 
 export function searchParamsToQueryString(params: SearchParams | null): string {
-  return `http://localhost:4000/movies?searchBy=${params?.searchBy || ''}&filter=${params?.filter || ''}&sortBy=${params?.sortBy || ''}&sortOrder=asc`;
+  return `http://localhost:4000/movies?searchBy=${params?.searchBy || ''}&filter=${params?.filter || ''}&sortBy=${params?.sortBy || 'id'}&sortOrder=desc`;
```

Comment:
utils can't have hardcoded values as base url `localhost...`, can't know about api keys as `sortBy` and `searchBy` 

utils are functions that can be used cross different projects. 

this util should get any object and return string based on its keys and values.